web/jest.config.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
 	transformIgnorePatterns: ["/node_modules/(?!(axios)/)", "/dist/"],
 	setupFilesAfterEnv: [
 		"./jest.presetup.js",
-		"./jest.setup.ts",
+		"./jest.setup.tsx",
```

Comment:
this can probably change back to ts if we are no longer rendering jsx in that mock?